backend/app/core/video_compose.py

```
@@ -0,0 +1,112 @@
+    Returns:
+        成功時: {"num_clips": int, "duration": float}
+        1つも合成できなかった場合: None
+        （空/失敗時の振る舞い—dict返却 vs 例外送出—は呼び出し側に委ねる）
+
+    NOTE: moviepy は重い依存なので関数内 import（import 副作用ゼロを維持）。
+          エンコード設定(fps=2/libx264/aac/2000k/ultrafast)は従来の固定値を踏襲。
+    """
+    from moviepy import ImageClip, AudioFileClip, concatenate_videoclips
+
+    clips = []
+    for i, (png_path, audio_path) in enumerate(zip(png_files, audio_files)):
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)